tests/plugins/wlm/query_group/query_group.yaml

```diff
@@ -0,0 +1,67 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
```

Comment:
Since the API is `/_wlm/query_group` this should be in `tests/default/wlm/query_group.yaml`.